Test/Core/Components/RigidBodyTest.cs

```diff
@@ -1,21 +1,54 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.IO;
-using System.Reflection;
-
-using Duality;
-using Duality.Cloning;
+﻿using System.Linq;
+using System.Threading;
 using Duality.Components;
 using Duality.Components.Physics;
-
+using Duality.Resources;
 using NUnit.Framework;
 
 namespace Duality.Tests.Components
 {
 	[TestFixture]
 	public class RigidBodyTest
 	{
+		[Test] public void FallingBallOnPlatform()
+		{
+			var scene = new Scene();
+			Scene.SwitchTo(scene);
+
+			//Create the ball
+			var ball = new GameObject("Ball");
+			var ballTransform = ball.AddComponent<Transform>();
+			var ballBody = ball.AddComponent<RigidBody>();
+			var listener = ball.AddComponent<CollisionListener>();
+			bool onCollisionBeginCalled = false, onCollisionEndCalled = false, onCollisionSolveCalled = false;
```

Comment:
See comment on collision listener test class, replace flags set in inline event handlers with list storage.